jsr.json

```
@@ -0,0 +1,15 @@
+{
+  "name": "@eslint/markdown",
+  "version": "6.0.0",
+  "exports": "./dist/esm/index.js",
+  "publish": {
+    "include": [
+      "dist/esm/index.js",
+      "dist/esm/index.d.ts",
+      "dist/esm/types.ts",
```

```suggestion
      "dist/esm/types.ts",
      "dist/esm/types.d.ts",
```

Seems that https://github.com/eslint/markdown/commit/c29d88da600f0a7b35d1514552f9cc107da6b8a3 was lost in rebase.